util/clientmetric/clientmetric.go

```diff
@@ -271,9 +271,16 @@ type deltaEncBuf struct {
 // writeName writes a "name" (N) record to the buffer, which notes
 // that the immediately following record's wireID has the provided
 // name.
-func (b *deltaEncBuf) writeName(name string) {
+func (b *deltaEncBuf) writeName(name string, typ Type) {
+	var name_prefix string
```

Comment:
Go doesn't really use underscores in variable names. They're super rare. https://go.dev/doc/effective_go#mixed-caps

Reply:
Brain fart. It'll take a little while longer for ~10 years of Python snake_case naming to wear off.